detachment/plugins/org.polarsys.kitalpha.model.common.scrutiny.contrib.viewpoints/src/org/polarsys/kitalpha/model/common/scrutiny/contrib/viewpoints/scrutinizes/UsedAFViewpoints.java

```diff
@@ -51,12 +52,14 @@
  */
 public class UsedAFViewpoints implements IScrutinize<ViewpointTreeContainer, Object> {
 
-	private Set<String> usedNsURI = new HashSet<String>();
-	private Set<String> usedViewpoints = new HashSet<String>();
+	private Set<String> usedNsURI = new HashSet<>();
+	private Set<String> usedViewpoints = new HashSet<>();
 	private ViewpointTreeContainer container;
 	private final ResourceHelper resourceReuseHelper = ResourceReuse.createHelper();
+	private static final String CAPELLA_CORE_VP_URI = "org.polarsys.capella.core.viewpoint";
```

Comment:
we shouldn't have any reference to capella

Reply:
Yes indeed. Thanks